projects/storyteller/webapp/src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,37 @@
+.home {
+    font-family: Satisfy;
+    font-size: 36px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: center;
+    color: #4e4e4e;
+}
+ 
+a:hover, a:active{
+    color: black;
+}
+ 
+.navigation-items{
+    list-style-type: none;
+}
```

Comment:
Do you know scss?
It's much more powerful and convenient than css.
https://sass-lang.com/documentation/file.SASS_REFERENCE.html
Try to google it

Reply:
Compare:

**css**
```css
.house {
  margin: 10px;
}
.roof {
  background: red;
  padding: 20px;
}
.walls {
  font-size: 14px;
}
.walls:hover {
  color: black;
}
```
**scss**
```scss
.house {
  margin: 10px;
  >.roof {
    background: red;
    padding: 20px;
  }
  >.walls {
    font-size: 14px;
    &:hover {
      color: black;
    }
  }
}
```